modules/exploits/linux/http/geutebruck_cmdinject_cve_2021_335xx.rb

```diff
@@ -38,7 +38,7 @@ def initialize(info = {})
           ['CVE', '2021-33552'],
           ['CVE', '2021-33553'],
           ['CVE', '2021-33554'],
-          [ 'URL', 'http://geutebruck.com' ],
+          [ 'URL', 'http://web.archive.org/web/20210226205340/https://www.geutebruck.com/' ],
```

Comment:
We'll want to double check that these new links are actually valid; it looks like this link was wrong from the get-go it should've been https://www.geutebrueck.com/index.html I believe